Features/Stories/FieldTypes/TextlineFieldType.feature

```diff
@@ -0,0 +1,246 @@
+
+    @javascript @common
+    Scenario: Creating a valid text line Field works when using a value within limited character scope
+        Given a Content Type with an "text line" Field exists with Properties:
+            | Validator                | Value |
+            | minimum length validator | 2     |
+            | maximum length validator | 4     |
+        When I create a content of this Type
+        And I set "LOL" as the Field Value
+        And I publish the content
+        Then the Content is successfully published
+
+    @javascript @edge
+    Scenario: Creating an invalid text line Field fails validation when using a value smaller than minimum character limit allowed
+        Given a Content Type with an "text line" Field exists with Properties:
+            | Validator                | Value |
+            | minimum length validator | 2     |
+            | maximum length validator | 4     |
+        When I create a content of this Type
+        And I set "X" as the Field Value
+        And I publish the content
+        Then Publishing fails with validation error message "The value should have at least 2 characters"
+
+    @javascript @edge @notWorking
+    Scenario: Creating an invalid content type with a text line Field fails validation when using a maximum bigger than the minimum character limit
+        Given I am on the "Content types" page
+        And I click on the "Content" link
+        When I click at "Create a content type" button
+        And I fill form with:
+            | Field      | Value |
+            | Name       | Test  |
+            | Identifier | test  |
+        And I add a field type "Text line" with:
+            | Field           | Value |
+            | Name            | Text  |
+            | Identifier      | text  |
+            | Minimum length  | 4     |
+            | Maximum length  | 2     |
+        And I click at "OK" button
+        Then Publishing fails with validation error message "Form did not validate. Please review errors below."
+
+    @javascript @edge @notWorking
+    Scenario: Creating a valid content type with a text line Field works when using a minimum equal to the maximum character limit
+        Given I am on the "Content types" page
+        And I click in the "Content" Content type group
+        When I click at "Create a content type" button
+        And I fill form with:
+            | Field      | Value |
+            | Name       | Test  |
+            | Identifier | test  |
+        And I add a field type "Text line" with:
+            | Field           | Value |
+            | Name            | Text  |
+            | Identifier      | text  |
+            | Minimum length  | 4     |
+            | Maximum length  | 4     |
+        And I click at "OK" button
+        Then the Content is successfully published with the message "The Content type draft was successfully updated and published. Related Content has also been updated."
```

Comment:
This scenario publishes a Content Type, not a Content. Also, in this case, I think you should not test the message.